docs: Admin training (#2436)
caos/zitadel: docs/docs/concepts/zitadel/objects/projects.md

<ProjectDescription name="ProjectDescription" />

## Project Settings

On default the login screen will be shown in the private labeling settings of the system (e.g zitadel.ch).
With the scope primary domain [scope](../../../apis/openidoauth/scopes#reserves-scopes) it is possible to trigger the setting of the given organization. 
But this will also restrict, the login to user of the given organization.  

With the private labeling setting it is possible to choose which settings should trigger.

| Setting | Description |
| --- | --- |
| Unspecified | If nothing is specified the default will trigger. (System settings zitadel.ch) |
| Enforce project resource owner policy | This setting will force the private labeling of the organization (resource owner) of the project setting. |
| Allow Login User resource owner policy | With this setting first the private labeling of the organization (resource owner) of the project will trigger. As soon as the user is identified and ZITADEL know to which organization (resource owner) he belongs, the settings of the organization of the user will trigger. |

## Applications

Applications define the different clients, that share the same roles. 

> For the moment only OIDC clients are supported. Soon JWT and SAML will follow.

Reviewer: ```suggestion
At the moment we support OIDC and almost every OAuth2 client. We'll be expanding this with SAML shortly.
```

Go to [Applications](./applications) for more details.

## Granted Organizations

To enable another organization to use a project, the organization needs a grant to the project.
Only the selected roles will be available to the granted organization.

> The granted organization will be able to manage the authorization to his users for the granted project by himself in his own organization.

Reviewer: ```suggestion
The granted organization will be able to manage the authorizations of his users for the granted project by himself in his own organization.
```

More about granted projects: [Granted Projects](./granted_projects)

## Roles

> A role consists of different attributes. For the authorization only the key is relevant, which must therefore be unique.

Reviewer: ```suggestion
A role consists of different attributes. Only the key is relevant to the authorization and must therefore be unique.
```
